### DISCUSSION TITLE
Bike robot

### DISCUSSION THREAD

**Discussion by Gloamer97223** [AtsushiSakai/PythonRobotics#786] [General]
'_'I would like to build  a  Robot for my rear bike that turns "head" toward approaching cars, and emits a doppler rising tone on the sides, one for direction, both for rear..  I know a little  about Python, so I'd like to program it in that.  
Mountable on back of my e-bike, <10 lbs.  I'm willing to pay reasonable money for parts for something this customized.  Being able to use the bike batteries would be a plus.  Want to build it before Halloween!  I'll add a mike connection to it from my voice to warn people not seeing me.  Or people that did see  me but chose to ignore my bicycling rights. :(

**Comment by ayushcodec** [comment]
can you please give a little more detailed description of the project with some refereces
